<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratePoints_Dialog</class>
 <widget class="QDialog" name="GeneratePoints_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>191</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Generate points</string>
  </property>
  <widget class="QPushButton" name="_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Spacing Options</string>
   </property>
   <widget class="QRadioButton" name="_102">
    <property name="geometry">
     <rect>
      <x>-90</x>
      <y>20</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>By spacing        </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="_103">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>40</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>By quantity       </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of points</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Spacing</string>
    </property>
   </widget>
   <widget class="RCDoubleSpinBox" name="_104">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
   </widget>
   <widget class="RCIntegerSpinBox" name="_105">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>105</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RCDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../RealCADQt/source/rcdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>RCIntegerSpinBox</class>
   <extends>QSpinBox</extends>
   <header>../RealCADQt/source/rcintegerspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_104</tabstop>
  <tabstop>_105</tabstop>
  <tabstop>_1</tabstop>
  <tabstop>_2</tabstop>
  <tabstop>_102</tabstop>
  <tabstop>_103</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
